rastervision_core/rastervision/core/rv_pipeline/semantic_segmentation_config.py

```diff
@@ -61,6 +61,17 @@ class SemanticSegmentationConfig(RVPipelineConfig):
     chip_options: SemanticSegmentationChipOptions = SemanticSegmentationChipOptions(
     )
 
+    img_channels: int = Field(
+        3, description='The number of channels of the training images.')
+
+    channel_display_groups: Optional[Union[dict, list, tuple]] = Field(
```

Comment:
A little more information in the description would be good since the docs are generated from this. Specifically what the content of the dict or list or tuple should contain.

Reply:
Done. It now reads:
>Groups of image channels to display together as a subplot when plotting the data and predictions. Can be a list or tuple of groups (e.g. [(0, 1, 2), (3,)]) or a dict containing title-to-group mappings (e.g. {"RGB": [0, 1, 2], "IR": [3]}), where each group is a list or tuple of channel indices and title is a string that will be used as the title of the subplot for that group.